ArduCopter/crash_check.cpp

```diff
@@ -23,6 +23,12 @@ void Copter::crash_check()
         return;
     }
 
+    // exit immediately if in standby
+    if (copter.standby_active == true) {
```

Comment:
we can remove the "copter." and "== true" parts from this "if" and a few lower down because standby_active is already a boolean and this method is in the Copter class.

Reply:
Cool, done.